docs/guides/divisions.mdx

```diff
@@ -2,18 +2,367 @@
+└──────────────┴──────────────┘
+```
 
-- A `division` is a feature type that represents an official or non-official organization of people-- country, region, province, city, neighborhood, etc.-- as seen from a given political perspective. It has a `Point` geometry which gives an approximate location of the position most commonly associated with the feature.
 - A `division_area` is a feature type that captures the shape of the land area, or land and territorial sea (maritime), belonging to a `division` feature. It has a `Polygon` or `MultiPolygon` geometry.
+
+```
+┌──────────────┬──────────────┐
+│ count_star() │   subtype    │
+│    int64     │   varchar    │
+├──────────────┼──────────────┤
+│       540557 │ locality     │
+│       385886 │ neighborhood │
+│        36973 │ county       │
+│        18470 │ localadmin   │
+│         3866 │ region       │
+│          272 │ country      │
+└──────────────┴──────────────┘
+```
+
 - A `boundary` is a feature type that represents a shared border between two `division` features. It has a `LineString` geometry. The geometry of a `boundary` is either wholly non-maritime, or wholly maritime. A maritime boundary is the extension of a non-maritime boundary into the water.
 
-## Schema reference
+```
+┌──────────────┬─────────┐
+│ count_star() │ subtype │
+│    int64     │ varchar │
+├──────────────┼─────────┤
+│        64107 │ county  │
+│         7116 │ region  │
+│          373 │ country │
+└──────────────┴─────────┘
+```
+
+### Table of Contents
+
+<details>
+	<summary>Divisions Description</summary>
+	1. [Schema Reference](#schema_reference)
+	2. [Sources and Licensing](#sources_and_licensing)
+	3. [Pipeline Migration to Overture](#pipeline_migration_to_overture)
+	4. [Subtypes](#subtypes)
+	5. [Data Fields](#data_fields)
+</details>
+
+<details>
+	<summary>Data Access and Retrieval</summary>
+	1. [Data Retrieval Examples](#data_retrieval_examples)
+		1. [SQL](#sql)
+		2. [Python](#python)
+</details>
+
+<details>
+	<summary>Contact</summary>
+	1. [Feedback](#feedback)
+	2. [Discussions](#discussions)
+	3. [Issues](#issues)
+</details>
+
+<details>
+	<summary>Revision History</summary>
+	1. [Version Info](#version_info)
+	2. [GERS Changelog](#gers_changelog)
+</details>
+
+
+# Divisions Description
+
+### Schema reference <a name="schema_reference"></a>
 
 - [Explore the schema reference for the `division_boundary` feature type](/schema/reference/divisions/division_boundary).
 - [Explore the schema reference for the `division` feature type](/schema/reference/divisions/division).
 - [Explore the schema reference for the `division_area` feature type](/schema/reference/divisions/division_area).
+
+### Sources and Licensing <a name="sources_and_licensing"></a>
+
+As of the July 2024 release, Overture's divisions theme is comprised of data from the following sources:
+
+- [OpenStreetMap](https://openstreetmap.org/) ([ODbL-1.0 license terms]("https://www.openstreetmap.org/copyright"))
```

Comment:
I think it's the `"` around the links here that's breaking the build

Reply:
Thanks - that seems to have fixed the build issues.